network/metadata.csv

```diff
@@ -49,6 +49,64 @@ system.net.packets_in.error,gauge,,error,second,The number of packet receive err
 system.net.packets_out.count,gauge,,packet,second,The number of packets of data transmitted by the interface.,0,system,packets out
 system.net.packets_out.drop,gauge,,packet,second,The number of packet transmit drops detected by the device driver.,-1,system,pkts out drop
 system.net.packets_out.error,gauge,,error,second,The number of packet transmit errors detected by the device driver.,-1,system,pkts out err
+system.net.ip.in_receives,gauge,,datagram,,Number of received IP datagrams (including dropped datagrams) (Linux only).,0,system,ip rcv
```

Comment:
nit- some of the short names are duplicated, can you make them unique?

Reply:
should be fixed